setup.py

```diff
@@ -112,7 +104,7 @@ def install_needed_req(import_name, package_name=None, min_version=None, max_ver
 setup(
     name=PACKAGE_NAME,
     version=VERSION,
-    packages=setuptools.find_namespace_packages(include=['qiskit.*']),
+    packages=setuptools.find_packages(include=['qiskit_aer.*']),
```

Comment:
I guess that this copies packages in `qiskit_aer/` but not files in `qiskit` (`qiskit_aer/__init__.py`, `qiskit_aer/aererror.py`, `qiskit_aer/aerprovider.py` and `qiskit_aer/version.py`).